config/install/core.extension.yml

```diff
@@ -0,0 +1,96 @@
+module:
+  admin_toolbar: 0
+  admin_toolbar_links_access_filter: 0
+  admin_toolbar_tools: 0
+  allowed_formats: 0
+  anchor_link: 0
+  big_pipe: 0
+  block: 0
+  breakpoint: 0
+  chosen: 0
+  chosen_lib: 0
+  ckeditor: 0
+  components: 0
+  config: 0
+  config_distro: 0
+  config_filter: 0
+  config_merge: 0
+  config_split: 0
+  config_update: 0
+  contextual: 0
+  crop: 0
+  ctools: 0
+  datetime: 0
+  datetime_range: 0
+  default_content: 0
+  dropzonejs: 0
+  dropzonejs_eb_widget: 0
+  dynamic_page_cache: 0
+  editor: 0
+  embed: 0
+  entity_browser: 0
+  entity_browser_entity_form: 0
+  entity_embed: 0
+  entity_reference_revisions: 0
+  entity_usage: 0
+  externalauth: 0
+  fakeobjects: 0
+  field: 0
+  field_ui: 0
+  file: 0
+  filter: 0
+  focal_point: 0
+  hal: 0
+  help: 0
+  image: 0
+  inline_entity_form: 0
+  inline_form_errors: 0
+  jumpstart_ui: 0
+  layout_builder: 0
+  layout_discovery: 0
+  link: 0
+  linkit: 0
+  media: 0
+  media_duplicate_validation: 0
+  menu_block: 0
+  menu_ui: 0
+  nobots: 0
```

Comment:
Perhaps leave nobots out of the core install and leave it up to the environment to manage.

Reply:
i put it into there since we arent going to turn nobots off until RC1. we could move it then

Reply:
Ok, I can get on board with that, but one last question. Won't we want to have this as part of our environment workflow stuff that we are figuring out anyway? eg: Enable/Disable on specific environments?

Reply:
in the `acsf-cardinalsites` PR there is enable/disable on local vs ACSF so it would be an easy move for now... but when we get to the point of RC1 we dont plan to use config management so then this config will be removed at that time anyways

Reply:
I would vote to put this logic in Apache.

Reply:
Nobots was written for users who do not have access to Apache config and only to the Drupal GUI.

Reply:
but we'll also need nobots for non-production sites on the production. It still gonna be needed

Reply:
Good call. We want the module there but disabled. General logic for `dev/test/prod` in `.htaccess`. User-controlled `nobot` for `prod`.